docs/release_notes/release-2.3.md

```diff
@@ -1,5 +1,39 @@
 # StarRocks version 2.3
 
+## 2.3.16
+
+Release date: August 4, 2023
+
+### Bug Fixes
+
+Fixed the following issue:
+
+FE memory leak caused by blocked LabelCleaner threads. [#28311](https://github.com/StarRocks/starrocks/pull/28311) [#28636](https://github.com/StarRocks/starrocks/pull/28636)
+
+## 2.3.15
+
+Release date: July 31, 2023
+
+## Improvements
+
+- Optimized tablet scheduling logic to prevent tablets remaining pending for a long period or an FE crashing under certain circumstances. [#21647](https://github.com/StarRocks/starrocks/pull/21647) [#23062](https://github.com/StarRocks/starrocks/pull/23062) [#25785](https://github.com/StarRocks/starrocks/pull/25785)
+- Optimized the scheduling logic of TabletChecker to prevent the checker from repeatedly scheduling tablets that are not repaired. [#27648](https://github.com/StarRocks/starrocks/pull/27648)
+- The partition metadata records visibleTxnId, which corresponds to the visible version. When versions of replicas for a tablet are not consistent, it is easier to trace the transaction that created this version . [#27924](https://github.com/StarRocks/starrocks/pull/27924)
```

Comment:
1. consistent between who?  suggest: The partition metadata records visibleTxnId, which corresponds to the visible version of tablet replicas. When the replica version of a tablet on the BE is inconsistent with that on the FE, xxx      2. extra space at the end of this sentence